renderdoc/driver/vulkan/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+set(sources
+    vk_common.cpp
+    vk_common.h
+    vk_core.cpp
+    vk_core.h
+    vk_counters.cpp
+    vk_debug.cpp
+    vk_debug.h
+    vk_dispatchtables.cpp
+    vk_dispatchtables.h
+    vk_hookset_defs.h
+    vk_info.cpp
+    vk_info.h
+    vk_initstate.cpp
+    vk_manager.cpp
+    vk_manager.h
+    vk_memory.cpp
+    vk_replay.cpp
+    vk_replay.h
+    vk_resources.cpp
+    vk_resources.h
+    vk_state.cpp
+    vk_state.h
+    vk_tracelayer.cpp
+    official/vk_layer.h
+    official/vk_lunarg_debug_marker.h
+    official/vk_platform.h
+    official/vulkan.h
+    wrappers/vk_cmd_funcs.cpp
+    wrappers/vk_descriptor_funcs.cpp
+    wrappers/vk_device_funcs.cpp
+    wrappers/vk_draw_funcs.cpp
+    wrappers/vk_dynamic_funcs.cpp
+    wrappers/vk_get_funcs.cpp
+    wrappers/vk_misc_funcs.cpp
+    wrappers/vk_queue_funcs.cpp
+    wrappers/vk_resource_funcs.cpp
+    wrappers/vk_shader_funcs.cpp
+    wrappers/vk_sync_funcs.cpp
+    wrappers/vk_wsi_funcs.cpp)
+
+set(definitions ${RDOC_DEFINITIONS})
+
+if(WIN32)
```

Comment:
Same as above - since Win32 isn't supported for cmake probably better not to have this code. It will maybe cause confusion to have it, and is at risk of becoming out of date if e.g. the json build process changes.